### DISCUSSION TITLE
Happy 7th Anniversary VisiData!

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#2095] [❤️]
Congratulations on the big day!  I've been a major fan for at least 4.5 years.  I'm excited to see the shape of the upcoming 3.0 release. Also, I'm so happy to see the community grow over these last few years with new contributors and participants.

There was a great blog post on http://visidata.org on the 5th birthday of VisiData in 2021.

Thanks, @saulpw!  And everyone who has helped make VisiData wonderful.

**Comment by saulpw** [comment]
Thanks, @frosencrantz! I'm also really excited for 3.0 and where VisiData is headed, and it's so heartening to have you and everyone here contributing to this standout community and tool.  Here's to another year!
